src/main/webapp/resources/js/pages/user/components/UserTable.jsx

```diff
@@ -56,7 +56,7 @@ export function UserTable() {
       fixed: "left",
       render(text, full) {
         return IS_ADMIN ? (
```

Comment:
A anchor tag causes a full page reload, you need to you the `Link` component provided by react-router

Reply:
I made a couple more changes, because document.referrer wasn't working for me anymore. Please see 3f60e2b8bbe9a05d6c1071af053fc660257f3ac2.